.github/workflows/elliptic-curve.yml

```diff
@@ -50,6 +50,7 @@ jobs:
       - run: cargo build --target ${{ matrix.target }} --release --no-default-features --features pkcs8
       - run: cargo build --target ${{ matrix.target }} --release --no-default-features --features sec1
       - run: cargo build --target ${{ matrix.target }} --release --no-default-features --features serde
+      - run: cargo build --target ${{ matrix.target }} --release --no-default-features --features alloc,serde
```

Comment:
I'm sure that there are other combinations we should test ... is this alright or should I just remove this?

Reply:
We should probably switch to `cargo hack` to test all combinations. This is fine for now, though.

Reply:
Do not forget about combinatorial explosion. I think it's worth to limit execution time of CI jobs by several minutes.

Reply:
The surface of each feature is pretty small. The features mostly exist for gating dependencies, and compilation results for those should be reused across runs